database/youtube/data-structures.json

```
@@ -1,65 +1,74 @@
-[{
+[
+  {
     "name": "Jennys Lectures CSIT",
     "description": "This channel provides free courses for DSA, C, C++, Python etc.",
     "url": "https://www.youtube.com/@JennyslecturesCSIT",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-},
-{
+  },
+  {
     "name": "CodeHelp - by Babbar",
     "description": "This channel provides free placement specific DSA course in C++.",
     "url": "https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Apna College",
     "description": "This channel provides free courses for DSA, Java, C++, C etc.",
     "url": "https://www.youtube.com/@ApnaCollegeOfficial",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "CodeWithHarry",
     "description": "A all in one destination for DSA, Python, C++, JavaScript, Nodejs, Java, PHP & many more.",
     "url": "https://www.youtube.com/@CodeWithHarry/playlists",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Luv",
     "description": "This channel provides free Competitive Programming content.",
     "url": "https://www.youtube.com/@iamluv/playlists",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "take U forward",
     "description": "One of the best channel for DSA, you will find the thought process behind every problem and not just simple algorithms.",
     "url": "https://www.youtube.com/@takeUforward/featured",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-},
-{
+  },
+  {
     "name": "Pepcoding",
     "description": "One of the best channel for DSA in Java and helpful for both students and working professionals.",
```

[Proofreading] see [my feedback on line 44](https://github.com/rupali-codes/LinksHub/pull/587/files#r1199674921). Also, does this channel teach DSA and Java? If so, replace "in" with "and" to clarify this. 
```suggestion
    "description": "One of the best channels for DSA and Java. It is also helpful for both students and working professionals.",
```